### DISCUSSION TITLE
Pinia store not populating with fresh session

### DISCUSSION THREAD

**Discussion by tmoreland72** [vuejs/vuefire#1300] [Questions and Help]
I'm running into a problem that I just can't figure out.  When I open my app and login my pinia stores are initialized but empty.  The firestore refs all require a "tenantId" value for the where clause.  If I reset my browser, everything gets populated.  If I logout and login again, everything is populated.  If I logout, reset browser, and login, the stores aren't populated.  

The "tenantId" is stored in `app-store` as part of the login process.  

When I log out I can see that the stores are reset to initial state.  I also tried to dispose them but same outcome.

app-store:

```js
import { defineStore } from 'pinia'
import { SessionStorage } from 'quasar'

export const useAppStore = defineStore('app', {
   state: () => {
      return {
         userId: SessionStorage.has('userId')
            ? SessionStorage.getItem('userId')
            : '',
         tenantId: SessionStorage.has('tenantId')
            ? SessionStorage.getItem('tenantId')
            : '',
         sessionId: SessionStorage.has('sessionId')
            ? SessionStorage.getItem('sessionId')
            : '',
         preferences: {},
         settings: {},
         loading: false
      }
   },
})
```

aggregates-store:

```js
import { collection, query, where } from 'firebase/firestore'
import { useCollection, useFirestore } from 'vuefire'
import { defineStore } from 'pinia'

import { useAppStore } from 'stores/app-store'

const db = useFirestore()
const $app = useAppStore()

export const useAggregatesStore = defineStore('aggregates', () => {
   const tenantId = $app.tenantId
   console.log('aggregatesStore', 'tenantId', tenantId)
   const dbRef = query(
      collection(db, 'aggregates'),
      where('tenantId', '==', tenantId)
   )
   const all = useCollection(dbRef)
   return { all }
})
```

Make sense??

**Comment by posva** [comment]
If `$app.tenantId` is reactive, you would need to make `dbRef` a computed property. Other than that, I don't see what could be failing.
